### PR TITLE
docs: replace the support library packages with androidx

### PR DESCRIPTION
**Pull request by rosariopfernandes**
I've double-checked to make sure we're indeed using these androidx libraries:

https://github.com/firebase/FirebaseUI-Android/blob/6de2d73aad174b3011912b1170646e2d4f01dbf9/buildSrc/src/main/kotlin/Config.kt#L39

https://github.com/firebase/FirebaseUI-Android/blob/6de2d73aad174b3011912b1170646e2d4f01dbf9/buildSrc/src/main/kotlin/Config.kt#L34

https://github.com/firebase/FirebaseUI-Android/blob/6de2d73aad174b3011912b1170646e2d4f01dbf9/buildSrc/src/main/kotlin/Config.kt#L40

### CODE REVIEW

**Comment by samtstern** [comment]
@rosariopfernandes I just changed the target from `master` to `version-6.3.0-dev` so you might have to rebase.

**Comment by rosariopfernandes** [comment]
@samtstern done! Not sure why the proguard-tests keep failing :thinking:

File: database/README.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ The `FirebaseRecyclerPagingAdapter` is built on top of the [Android Paging Suppo
 Before using the adapter in your application, you must add a dependency on the support library:
 
 ```groovy
-implementation 'android.arch.paging:runtime:1.x.x'
+implementation 'androidx.paging:paging-runtime:2.x.x'
```
Comment (samtstern) [inline database/README.md:254]: Is the part of the paging library (`PagedList.Config`) that we use unchanged from 1.x to 2.x?

Comment (rosariopfernandes) [reply database/README.md:254]: @samtstern Yes, `PagedList.Config` remains unchanged.

**Comment by samtstern** [comment]
@rosariopfernandes I worked out the build issues!

**Comment by rosariopfernandes** [comment]
@samtstern amazing! :)

**Pull request merged**